charts/codesealer/templates/NOTES.txt

```diff
@@ -1,5 +1,19 @@
+{{- if .Values.manager.enabled }}
+The first time core-manager is started, it will create a Maintenance Administrator. 
+To see the credentials for this account, check the core-manager log: 
+
+  $ kubectl get pods -n codesealer-system
+  $ kubectl logs <core-manager-pod> -n codesealer-system | grep password
+
+The manager can be accessed at https://localhost
```

Comment:
Is this true without any port forwarding? If so, should it be?

Reply:
Yes. This works on my machine.  Seeing some Ingress issues on Felipe's machine which we are debugging.  I usually see this as http://localhost:8080 with port forwarding.  If you think that is better I can move to that.

Reply:
I moved to http://localhost:8444